tools/pipelines/build-client.yml

```diff
@@ -152,10 +159,10 @@ extends:
     - webpack
     - ci:test:mocha
     - ci:test:jest
-    - ${{ if eq(variables['System.TeamProject'], 'public') }}:
+    - ${{ if or(eq(variables['System.TeamProject'], 'public'), eq(parameters.reducedCompatMatrix, true)) }}:
       - ci:test:realsvc:local
       - ci:test:realsvc:tinylicious
-    - ${{ if ne(variables['System.TeamProject'], 'public') }}:
+    - ${{ else }}:
```

Comment:
The only reason these were added because we shipped some serious bugs. I don't remember how bad they were, but it was a bad customer experience. It specifically tests loader and driver compat. I would recommend you move these tests to the e2e test pipelines as they aren't run there.

Reply:
That seems reasonable, I'll look into that. My take is that we can probably remove them here if we move them, since we do monitor the e2e tests pipeline closely to keep tinylicious test pass rate at 100%; we would still get notified even if a single test started failing over there.